components/Order/models.rst

```diff
@@ -0,0 +1,170 @@
+Models
+======
+
+.. _component_order_model_adjustment:
+
+Adjustment
+----------
+
+**Adjustment** object represents an adjustment to the order's or order item's total.
+Their amount can be positive (charges - taxes, shipping fees etc.) or negative (discounts etc.).
+Adjustments have the following properties:
+
++-------------+-----------------------------------------+--------------------+
+| Property    | Description                             | Type               |
++=============+=========================================+====================+
+| id          | Unique id of the adjustment             | mixed              |
++-------------+-----------------------------------------+--------------------+
+| order       | Reference to Order                      | OrderInterface     |
++-------------+-----------------------------------------+--------------------+
+| orderItem   | Reference to OrderItem                  | OrderItemInterface |
++-------------+-----------------------------------------+--------------------+
+| label       | Type of the adjustment (e.g. "tax")     | string             |
++-------------+-----------------------------------------+--------------------+
+| description | e.g. "Clothing Tax 9%"                  | string             |
++-------------+-----------------------------------------+--------------------+
+| amount      | Adjustment amount                       | int                |
++-------------+-----------------------------------------+--------------------+
+| neutral     | Boolean flag of neutrality              | bool               |
++-------------+-----------------------------------------+--------------------+
+| locked      | Adjustment lock (prevent from deletion) | bool               |
++-------------+-----------------------------------------+--------------------+
+| originId    | Origin id of the adjustment             | int                |
++-------------+-----------------------------------------+--------------------+
+| originType  | Origin type of the adjustment           | string             |
++-------------+-----------------------------------------+--------------------+
+| createdAt   | Date when adjustment was created        | \\DateTime         |
++-------------+-----------------------------------------+--------------------+
+| updatedAt   | Date of last change                     | \\DateTime         |
++-------------+-----------------------------------------+--------------------+
```

Comment:
I think we should remove the type from here. It is already in the api doc, when using IDE you have autocomplete etc. So this is a duplication, that we would need to maintain and keep up-to-date. WDYT?